### ISSUE TITLE
yum install python26 fails with no package available error

### ISSUE THREAD

**Issue by hessio** [Seagate/cortx-s3server#903]
Is it necessary to have python2 and python3 ?

`yum install python26` fails with no package available error

```
[root@cortx-deploy Python-2.6.6]# yum install python26
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: mirrors.gethosted.online
 * epel: d2lzkl7pfhq30w.cloudfront.net
 * extras: mirrors.gethosted.online
 * updates: mirrors.gethosted.online
No package python26 available.
Error: Nothing to do
```

Workaround I used:
```
wget https://www.python.org/ftp/python/2.6.6/Python-2.6.6.tgz
tar -zxvf Python-2.6.6.tgz
cd Python-2.6.6
./configure && make && make install
yum install python26

```

**Comment by nileshgovande** [comment]
@hessio - Can you please accomodate this change in https://github.com/Seagate/cortx-s3server/pull/900 ?
We don't need python 2.6.5 - anything above 2.7 will work. so pls replace 'yum install python26' with 'yum install python'

**Comment by hessio** [comment]
Resolved in PR #900

**Comment by nileshgovande** [comment]
Merged
